guestbook/README.md

```diff
@@ -47,7 +53,7 @@ $ kubectl cluster-info
 
 If you see a url response, you are ready to go. If not, read the [Getting Started guides](http://kubernetes.io/docs/getting-started-guides/) for how to get started, and follow the [prerequisites](http://kubernetes.io/docs/user-guide/prereqs/) to install and configure `kubectl`. As noted above, if you have a Google Container Engine cluster set up, read [this example](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
 
-All the files referenced in this example can be downloaded in [current folder](./).
+All the files referenced in this example can be downloaded [from GitHub](https://git.k8s.io/examples/guestbook).
 
```

Comment:
The actual content will be later cleaned up by @cody-clark later, so I'm just doing the bare minimum to keep the tutorial working.